config/routes.rb

```diff
@@ -76,77 +76,21 @@
     scope constraints: { type: /(published|draft|pending|expired|awaiting_feedback)/ } do
       get "jobs(/:type)", to: "hiring_staff/organisations#show", defaults: { type: :published }, as: :jobs_with_type
     end
-    resources :jobs, only: %i[new edit destroy delete show], controller: "hiring_staff/vacancies" do
-      get "review",
-          defaults: { create_step: 8, step_title: I18n.t("jobs.review_heading") }
+
+    resources :jobs, only: %i[create edit destroy delete show], controller: "hiring_staff/vacancies" do
+      resources :build, only: %i[show update], controller: "hiring_staff/vacancies/build"
+      resource :documents, only: %i[create destroy show], controller: "hiring_staff/vacancies/documents"
```

Comment:
why resource is singular here?

Reply:
Truthfully I don't know (they were always like that = copy 🍝 ) But after reading [this](https://rubyinrails.com/2019/04/16/rails-routes-difference-between-resource-and-resources/) I still think the singular resource is suitable here